examples/example_plugin/example_plugin/templates/example_plugin/example_with_custom_template.html

```diff
@@ -0,0 +1,4 @@
+<script>
+    console.log("i'm injecting js");
+</script>
+{% extends 'extras/job.html' %}
```

Comment:
@jathanism My Django-templating-foo is a bit rusty. Should we move the extends up to the top here?

Also are there sufficient `{% block %}`'s in the base job template to allow users to target sections of the form specifically?

Reply:
^ i've tested this manually and it worked, if you want to do with blocks {% block javascript %} {% block css %} we could do it that way but i haven't seen that done in the project elsewhere. (with my very limited experience of the project)

Reply:
👍 If the blocks don't exist already, that's fine but I think we should have the example plugin demo the cleanest/ideal use case.